app/components/hiring_staff/sidebar_component.html.haml

```diff
@@ -4,15 +4,15 @@
 
 #step-by-step-navigation.app-step-nav.app-step-nav--active
   %ol.app-step-nav__steps
-    - steps_to_display.each do |step|
-      %li.app-step-nav__step.js-step{ class: "#{set_active_step_class(step[:number])} #{set_visited_step_class(step[:number], @vacancy)}", 'aria-current': 'step', data: { show: '' } }
+    - steps_to_display.uniq { |_, v| v }.to_h.each do |step_name, step_values|
```

Comment:
Wouldn't this do the trick here as it never uses `step_name`?
```ruby
steps_to_display.values.each do |step_values|
```

Reply:
Classic case of completely changing something and never revisiting the other thing once I changed it...